.github/workflows/test.yml

```diff
@@ -13,9 +16,8 @@ jobs:
     strategy:
       matrix:
         version:
-          - '1.0'
```

Comment:
If we drop testing for 1.0 maybe one should also drop it's support?

Reply:
Yes, good point. I will tag a release with the `setwith!` first though (because why not).

Reply:
Or maybe I won't. Stuff it. Lol